sdk/react/ooyalaSkinBridgeListener.js

```diff
@@ -338,6 +340,19 @@ class OoyalaSkinBridgeListener {
     this.core.handleControlsTouch();
   };
 
+  onPipToggle(e) {
+    Log.log("Received PiP Toggle: " + e.isPipActivated + e.isPipButtonVisible);
+    this.skin.setState({
+      isRootPipActivated: e.isPipActivated,
+      //isRootPipButtonVisible: e.isPipButtonVisible
+    });
+    if (e.isPipButtonVisible) {
+      this.skin.setState({
```

Comment:
each `setState` can call render, so it's better to do several changes at one moment

Reply:
I decided that visibility is not necessary for this event (onPipToggle) and deleted it.